units/crons_remote_five_minute.go

```diff
@@ -52,6 +52,7 @@ func (j *cronsRemoteFiveMinuteJob) Run(ctx context.Context) {
 		PopulateHostProvisioningConversionJobs(j.env),
 		PopulateHostRestartJasperJobs(j.env),
 		logGithubAPILimit(j.env),
+		populateIPAddressAllocatorJobs(j.env),
```

Comment:
I noticed this is in the 5m cron, how many IPs should we allocate? When are we charged for them (I noticed you said we aren't charged for idle ones, is it only when we have a host with it we are charged?)? Is there ever a concern we allocate too much?

Reply:
Since the IPAM pool is dedicated solely for Evergreen's use, we're allocating the entire IPAM pool. This is a whole /18 CIDR block, so 16,384 addresses (minus a couple I reserved for staging to use). The job will also no-op once the IPAM pool runs out of addresses, meaning it's basically done allocating the entire pool.

Evergreen doesn't really have to pay anything for its usage of these IPs. MongoDB purchased its own big block of IPs addresses to use, so the cost has already been paid in advance for the IP addresses. Using those addresses in EC2 should also be free because we're using [the free tier of IPAM with bring-your-own-ip](https://aws.amazon.com/vpc/pricing/).

Reply:
Ah got it! Thank you for the context, it makes more sense that it's a temporary job then. Ty!